libsplinter/src/service/scabbard/mod.rs

```diff
@@ -201,6 +201,7 @@ impl Service for Scabbard {
         &mut self,
         service_registry: &dyn ServiceNetworkRegistry,
     ) -> Result<(), ServiceStopError> {
+        debug!("Stopping scabbard");
```

Comment:
I'd include the service ID here (for when there's more than once scabbard service running).

Reply:
updated